src/prefect/cli/work_pool.py

```diff
@@ -299,6 +301,42 @@ async def clear_concurrency_limit(
         exit_with_success(f"Cleared concurrency limit for work pool {name!r}")
 
 
+@work_pool_app.command()
+async def get_default_base_job_template(
```

Comment:
@desertaxle I gave about 30 seconds of thought to naming this, so could use some advice on a better name.

Reply:
Yeah, this is tough to name. My first thought is to nest a `base-job-template` sub command under `work-pool` so that the command would be `prefect work-pool base-job-template get-default --type=kubernetes`, but it feels premature to start adding subcommands. 

I'm OK with this if we don't think of anything else. Migrating this to another command isn't too bad, so we can wait until there are more commands and a clear grouping emerges.